hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/pbj/PbjConverter.java

```diff
@@ -1359,6 +1363,54 @@ public static com.hederahashgraph.api.proto.java.FileID fromPbj(FileID someFileI
                 .build();
     }
 
+    public static com.hederahashgraph.api.proto.java.CustomFee fromPbj(CustomFee customFee) {
+        var builder = com.hederahashgraph.api.proto.java.CustomFee.newBuilder();
+        if (customFee.hasFixedFee()) {
+            builder.setFixedFee(fromPbj(customFee.fixedFee()));
+        } else if (customFee.hasFractionalFee()) {
+            builder.setFractionalFee(fromPbj(customFee.fractionalFee()));
+        } else if (customFee.hasRoyaltyFee()) {
+            builder.setRoyaltyFee(fromPbj(customFee.royaltyFee()));
+        }
+
+        builder.setFeeCollectorAccountId(fromPbj(customFee.feeCollectorAccountId()));
+        builder.setAllCollectorsAreExempt(customFee.allCollectorsAreExempt());
+
+        return builder.build();
+    }
+
+    public static RoyaltyFee fromPbj(com.hedera.hapi.node.transaction.RoyaltyFee royaltyFee) {
```

Comment:
I believe we've been trying to consistently import the hapi classes and use the fully-qualified name only for the obsolete classes.